app/assets/stylesheets/common/base/compose.scss

```diff
@@ -1,3 +1,12 @@
+:root {
+  --composer-height: 300px;
+  --composer-height-title: 400px;
```

Comment:
I think we can set just one custom property here, and then add `100px` to it for the "New Topic" composer that has a title (and category/tags dropdowns). We do something similar (although the other way around) for the minimum height: https://github.com/discourse/discourse/blob/a8666bd01a808a8a4d2f93bd9b713c914d75d9fb/app/assets/stylesheets/common/base/compose.scss#L551

Reply:
Do you mean something like `calc(var(--composer-height) + 100px)`? That does not work because when you adjust the composer height, you basically start from the current height that will instantaneously set `--composer-height` to `--composer-height + 100px`.

I could probably make it work with scrolling, by subtracting the added height before updating the property.

Reply:
Hmm... now I realize that would a bit more complexity because then you'd have to add the 100px to the padding-bottom too.

Reply:
Yes, you're right, it's not easy to use one property. 

Let's go with the current implementation. 

Can I ask for one small naming change? I think it would be better to use: 

```
--reply-composer-height
--new-topic-composer-height
```

instead of 

```
--composer-height
--composer-height-title
 ```